### ISSUE TITLE
roadmap: kd.io first working version

### ISSUE THREAD

**Issue by rjeczalik** [koding/koding#9888]
### Roadmap

| command | tracking | assignee | due to |
| ----------| --------- | -------- | ------- |
| kd auth register | #9644 | @mehmetalisavas | ~5 Dec 2016~ |
| kd credential | #9640 | @rjeczalik| ~21 Nov 2016~ |
| kite auth with team info | #9926 | @rjeczalik | ~7 Dec 2016 (#9927)~ |
| kd auth login | #9643 | @rjeczalik | ~7 Dec 2016~ ~(blocked by #9926)~ 15 Dec 2016 (#10003 #10021 ...)  |
| kd stack | #9641 | @rjeczalik | ~17 Dec 2016~ |
| kd team list/ls | #9689 | @ppknap | ~1 Dec 2016~ (done by #9852) |
| kd machine list/ls | #9305 | @ppknap | ~28 Nov 2016~ (done by #9797) |
| kd machine mount list/ls/add/rm | #9306 #9481 | @ppknap  | ~6 Jan 2017~ ~30 Jan 2017~ 10 Feb 2017(done by #10531) |
| kd machine ssh | #9889 | @ppknap | ~13 Dec 2016~ (done by #10018 #10019) |
| kd machine run | #9890 | @rjeczalik  | ~21 Dec 2016~ 13 Mar 2017(done by #10740) |
| kd machine cp | #10615 | @ppknap | ~24 Feb 2017~ |
| kd.io homepage | #9894 | @usirin | ~7 Dec 2016~ |
| kd.io login page | #9895 | @usirin | ~9 Dec 2016~ |
| kd.io template details page | #9896 | @usirin | ~9 Dec 2016~ |
| kd.io user profile page | #9897 | @usirin | ~9 Dec 2016~ |
| kd.io user settings page | #9898 | @usirin | ~9 Dec 2016~ |
| kd daemon install | #9886 | @rjeczalik | ~21 Feb 2017~ |
| kd daemon rollback  | #9886 | @rjeczalik | ~22 Feb 2017~ |
| kd template prerequisites | #9813 | @rjeczalik | ~23 Feb 2017~ |
| kd template | #9642 | @rjeczalik | ~24 Feb 2017~ |
| kd template init | #9813 | @rjeczalik | ~12 Apr~ |
| kd init | #10985 | @rjeczalik | ~13 Apr~ |
| kd machine start/stop | #9252 | @rjeczalik  | ~17 Apr~ |

Please link your pieces for kd.io in this issue - for each feature please create/add an issue, where you're going to keep more fine-grained (if relevant) todo-list, so progress on a specific item can be seen there.

The dates are only for showing priorities, so if your work is blocked by any of the issue, you can see what priority it has - we can always reschedule.

If there are any blockers or dependencies between the issue, please list/describe them in the particular issues and not here.

If there exist any PRs that are connected to any of the tracking issues, please list them in those issues (and ideally in commit messages - like `Part of #xxxx.` or `Closes #xxxx.`).

**Comment by rjeczalik** [comment]
MVP completed, closing in favour of  https://github.com/koding/koding/projects/6.
